plugins/transport-grpc/src/test/java/org/opensearch/plugin/transport/grpc/ssl/SecureSettingsHelpers.java

```diff
@@ -110,7 +109,7 @@ static SecureAuxTransportSettingsProvider getSecureSettingsProvider(
     ) {
         return new SecureAuxTransportSettingsProvider() {
             @Override
-            public Optional<SSLContext> buildSecureAuxServerTransportContext(Settings settings, AuxTransport transport)
+            public Optional<SSLContext> buildSecureAuxServerTransportContext(Settings settings, String auxTransportSettingKey)
```

Comment:
Do you think `auxTransportType` or `transportType` would be a better name than `auxTransportSettingKey`, fe:
```suggestion
            public Optional<SSLContext> buildSecureAuxServerTransportContext(Settings settings, String auxTransportType)
```